cmd/amp/platform_pull.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// PlatformPull is the main command for attaching platform subcommands.
+var PlatformPull = &cobra.Command{
+	Use:   "pull",
+	Short: "Pull platform images",
+	Long:  `Pull all AMP platform images`,
+	Run: func(cmd *cobra.Command, args []string) {
+		pullAMPImages(cmd, args)
+	},
+}
+
+func init() {
+	PlatformPull.Flags().BoolP("silence", "s", false, "no console output at all")
+	RootCmd.AddCommand(PlatformPull)
+}
+
+func pullAMPImages(cmd *cobra.Command, args []string) error {
```

Comment:
I disagree, the --server (or the .amp.yaml config) is a configuration for all command of this binary. It should apply to the pull command. If I configure the CLI to manage an infra stack on a remote swarm manager, I don't want it to pull images locally.